src/components/common/ForeignJurisdiction.vue

```diff
@@ -121,6 +132,28 @@ export default class ForeignJurisdiction extends Mixins(CountriesProvincesMixin)
     this.selectedRegionName = val
   }
 
+  /** Validate country field */
```

Comment:
Moved field validation in the component

Reply:
I had only a quick look, and yes, this is a cleaner way for both the parent component and child component 👍